src/elements/header/spec/script.spec.js

```diff
@@ -0,0 +1,5 @@
+import header from '../script';
+
+test('foo is exported', () => {
+  expect(header.foo).toBeDefined();
+});
```

Comment:
I think if we add a snapshot example, I would make the example more meaningful. Maybe I'll do it once I add better modules to the boilerplate (or in other repo, like `accordion`) that actually contain some testable logic.